Makefile

```diff
@@ -102,12 +103,12 @@ install-tools:
 
 .PHONY: otelcol
 otelcol:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/$(GOOS)/otelcol $(BUILD_INFO) ./cmd/otelcol
+	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/$(GOOS)_$(GOARCH)/otelcol $(BUILD_INFO) ./cmd/otelcol
```

Comment:
The path to `otelcol` is likely used in a few other places in the readmes and other files. Please search globally and update everywhere.

Reply:
Will do.